### ISSUE TITLE
Validation/Test splits

### ISSUE THREAD

**Issue by mpalaourg** [li-plus/DSNet#21]
Hi there,

First, thanks for providing your code and boost the open-source mentality of video summarization research. 
I have a question about the sets of video used. In this [issue](https://github.com/li-plus/DSNet/issues/1) you are referring to a *validation set* `f_score`, but neither on the [splits](https://github.com/li-plus/DSNet/tree/master/splits) appear any `validation_keys` nor in the [source code](https://github.com/li-plus/DSNet/blob/master/src/anchor_free/train.py#L30) I see a validation set, but rather the use of the test set, as validation. Am I missing something, or the usage of a test set for picking the best model is data leakage?

I think that the proper usage is [1]:
* `Training set`:  The sample of data used to fit the model.
* `Validation set`: The sample of data used to provide an unbiased evaluation of a model fit on the training dataset while tuning model hyperparameters.
* `Test set`: The sample of data used to provide an unbiased evaluation of a final model fit on the training dataset.

To be honest, though, the small number of videos in the video summarization dataset, shouldn't be enough for specifying a `validation set`.

Thanks in advance.
George

[1]: https://towardsdatascience.com/train-validation-and-test-sets-72cb40cba9e7

**Comment by li-plus** [comment]
You are right, but the public video summarization datasets are really too small to have a test set. We validate and evaluate our model only on the validation set, following [VASNet](https://github.com/ok1zjf/VASNet), though it is still tiny. 
We really hoped to experiment on larger datasets, but there wasn't any at the time when this paper was published.
